pkgs/applications/graphics/emulsion/default.nix

```diff
@@ -15,7 +15,7 @@
 , libxcb
 , libxkbcommon
 , wayland
-, xdg_utils
+, xdg-utils
```

Comment:
Is that input unused?

Reply:
It appears so. I will make a separate commit.